BREAKING_CHANGES.md

```diff
@@ -1,11 +1,3 @@
 # Breaking Changes
 
-<!-- Add a header (##) with the version and list the breaking changes.
```

Comment:
I liked the explanations. What is the reasoning. to remove them?

Reply:
I was heavily inspired by other nventive open source repos such as this one https://github.com/nventive/MessageDialogService/blob/main/BREAKING_CHANGES.md?plain=1 where it has been cleared as well. However, I don't have a strong objection to bringing it back.